### DISCUSSION TITLE
Duda con test cases

### DISCUSSION THREAD

**Discussion by Crisax234** [IIC2133-PUC/2024-2#162] [Tarea 2] [answered]
Hola, tengo dos dudas sobre los test cases. La primera es que yo al comparar el archivo de mi output en VS Code, con el output esperado. Me aparece de la siguiente forma, donde el output mío es el mismo al esperado, pero al parecer hay un espacio en blanco que aparece. Intente eliminarlo desde mi código, pero no logro solucionarlo. Que puede estar causando esto?
![image](https://github.com/user-attachments/assets/dcdc2819-99c4-49cb-b4c3-cd2c3247e207)

Junto con la duda anterior, quería consultar también que herramientas se utilizan para comparar los archivos, ya que, tal vez, esto no sea un problema con esa herramienta en cuestión. 
Saludos!

**Comment by Crisax234** [comment]
Revisando los outputs que nos dan. Me di cuenta que el espacio en blanco esta incluido en el output esperado y no era mi output el que estaba agregando uno. Ahora cambio mi pregunta a si es que es necesario agregarlo o no se toma en cuenta.

**Reply by sebacarrascop** [reply, marked as answer]
hola! idealmente dejar los tests igual a como son entregados, igual en casos de que el error sea por añadir un espacio o un typo menor, siempre se dará un espacio en la recorrección donde se entregará puntaje total.

**Comment by dvillas28** [comment]
unrelated, pero como se llama esa extension para ver las diff con colores 👀 ?

**Reply by sebacarrascop** [reply]
si no me equivoco es la función build-in de vs code, haces click derecho a un archivo y poner “select for compare” y luego a otro y pones “compare with selected”
